packages/server/src/database/knex/seeds/insert-admin.js

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config()
+const bcrypt = require('bcryptjs')
+const ADMIN_EMAIL = process.env.ADMIN_EMAIL
+const ADMIN_PASSWORD = process.env.ADMIN_PASSWORD
```

Comment:
It seems like a good case for destructive assignments, you can do
```js
const { env: { ADMIN_EMAIL, ADMIN_PASSWORD } } = process 
```

Reply:
Ah thanks, I'm still not used to these :)